src/auth-service/services/mailer.js

```diff
@@ -17,6 +17,8 @@ const mailer = {
 
       const mailOptions = {
         from: constants.EMAIL,
+        name: constants.EMAIL_NAME,
```

Comment:
@FaithDaka 
The name is now a constant variable. I couldn't use ENV because of the spaces in the name.

Otherwise, please note that the emails still come in as "data" despite all our efforts! 😁😁

Happy to hear additional thoughts/feedback.

Thanks!

Reply:
Hi @FaithDaka , this has now been modified as per the learnings from another PR: https://github.com/airqo-platform/AirQo-api/pull/1119

You can now test out and share more feedback accordingly.

Thanks!